admin-docs/building-info-architecture.md

```diff
@@ -91,34 +92,47 @@ Step 1: Create a **CSV file** on your computer:
 
 Step 2: Populate the **CSV file** with the terms. Each term should occupy its own row. For example, here’s a sample CSV file viewed as plain text:
 
-```English,French,German```
+```
+English,French,German
+Man,Homme,Mann
+Woman,Femme,Frau
+Child,Enfant,Kind
+```
 
-```Man,Homme,Mann```
+And here’s the same file viewed in a spreadsheet program:
+![](images/image_19.png)
 
-```Woman,Femme,Frau```
+- Note: Avoid including duplicate values in the same column, as this will cause a validation error and prevent the import from working successfully.
 
-```Child,Enfant,Kind```
+You can group terms under a parent term by providing the child terms following the parent prefixed with a hyphen.
 
-And here’s the same file viewed in a spreadsheet program:
-![](images/image_19.png)
+```
+English,French,German
+Primary,Primaire,Primär
+-Red,-Rouge,-Rot
+-Green,-Vert,-Grün
+-Blue,-Bleu,-Blau
+No color,Sans couleur,Farblos
+```
 
-  - Note: Avoid including duplicate values in the same column, as this will cause a validation error and prevent the import from working successfully.
+- Note: the prefixing needs to be consistent across translations of a term and all grouped terms need to have a parent. Otherwise, the import will cause a validation error.
```

Comment:
I know this should have gone on the previous PR with the actual import, but I am not sure I agree this was the best approach for nesting values.  This is heavily dependent on the row order, which I don't think makes for a very flexible structure.

I would have liked a "in row" definition.  Something like
```
English,Spanish
Magenta,Magenta
Any Color,Cualquier Color
Primary|Red,Primario|Rojo
Cyan,Cian
Primary|Blue,Primario|Azul

Not sure, something that makes each row a "complete" concept without having to analyze the content of previous rows?